contracts/JBTiered721Delegate.sol

```diff
@@ -100,7 +100,7 @@ contract JBTiered721Delegate is IJBTiered721Delegate, JB721Delegate, Votes, Owna
     return
       JBIpfsDecoder.decode(
         store.baseUriOf(address(this)),
-        store.tierOfTokenId(address(this), _tokenId).data.tokenUri
+        store.tierOfTokenId(address(this), _tokenId).encodedIPFSUri
```

Comment:
Why not making _encodedIpfsUriOf externally accessible and just get store.encodedIpfsUriOf(..) (so we reduce the tierOfTokenId(..) return data size to what's needed)?

Reply:
good point!